Bug: Unable to build v2.29 maintenance branch

### Describe the bug.

When running the make commands on the v2.29 branch, I get the following errors:
```
Creating message catalogs
  Creating message catalog for anax
/tmp/anax-gopath/pkg/mod/k8s.io/apimachinery@v0.22.2/pkg/util/net/http.go:40:2: could not import golang.org/x/net/http2 (missing go.sum entry for module providing package golang.org/x/net/http2; to add:
	go mod download golang.org/x/net)
/tmp/anax-gopath/pkg/mod/golang.org/x/oauth2@v0.0.0-20200107190931-bf48bf16ab8d/internal/token.go:23:2: could not import golang.org/x/net/context/ctxhttp (missing go.sum entry for module providing package golang.org/x/net/context/ctxhttp; to add:
	go mod download golang.org/x/net)
/tmp/anax-gopath/pkg/mod/github.com/eclipse/paho.mqtt.golang@v1.3.5/netconn.go:26:2: could not import golang.org/x/net/proxy (missing go.sum entry for module providing package golang.org/x/net/proxy; to add:
	go mod download golang.org/x/net)
/tmp/anax-gopath/pkg/mod/k8s.io/client-go@v0.22.2/rest/request.go:36:2: could not import golang.org/x/net/http2 (missing go.sum entry for module providing package golang.org/x/net/http2; to add:
	go mod download golang.org/x/net)
gotext: extract failed: : : couldn't load packages due to errors: k8s.io/client-go/rest, golang.org/x/oauth2/internal, github.com/eclipse/paho.mqtt.golang and 1 more
make: *** [Makefile:503: i18n-catalog] Error 1
script returned exit code 2
```


### Describe the steps to reproduce the behavior.

Checkout Anax code and switch to v2.29 branch
Run: 
```
make temp-mod-version
make i18n-catalog
make 
```

### Expected behavior.

Build passes

### Screenshots.

_No response_

### Operating Environment

Linux amd64, arm64, armhf

### Additional Information

_No response_

--- a/go.mod
+++ b/go.mod
@@ -21,8 +21,9 @@ require (
 	github.com/stretchr/testify v1.7.0
 	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
 	golang.org/x/mod v0.5.1 // indirect
-	golang.org/x/net v0.0.0-20210805182204-aaa1db679c0d // indirect
-	golang.org/x/sys v0.0.0-20211013075003-97ac67df715c
+	golang.org/x/net v0.0.0-20220412020605-290c469a71a5 // indirect
+	golang.org/x/sys v0.0.0-20220412211240-33da011f77ad
+	golang.org/x/term v0.0.0-20220411215600-e5f449aeb171 // indirect
 	golang.org/x/text v0.3.7
 	golang.org/x/tools v0.1.7 // indirect
 	gopkg.in/alecthomas/kingpin.v2 v2.2.6

